pyoffers/utils.py

```diff
@@ -65,6 +65,23 @@ def prepend_model(self, value, model):
         return value
 
 
+class ValuesDict(dict):
+    """
+    Contains keys as containers. Returns value if container contains given key.
+    """
+
+    def __getitem__(self, key):
+        for k, v in self.items():
+            if key in k:
+                return v
+
+    def get(self, k, d=None):
+        return self.__getitem__(k) or d
```

Comment:
Would `return self[k] or d` work?

Reply:
oops, `ValuesDict` shouldn't even be here, we have removed it in previous pull request. This is a merge mistake.